resources/tools/mdsd/devops/pipeline/stages/impl/dockerized/maven/modules/Container/BuildInContainer.groovy

```diff
@@ -0,0 +1,11 @@
+lock("m2-cache-${CFG.slaveName}") {
+    sh "docker exec ${CFG.containerId} cp -r /.m2 /tmp"
+}
+sh "docker exec ${CFG.containerId} cp -r /ws /tmp"
+sh "docker exec ${CFG.containerId} mvn -Dbnd.home.dir=/tmp -s /settings.xml -f /tmp/ws/pom.xml clean verify"
+sh "docker cp ${CFG.containerId}:/tmp/ws/. ${CFG.workspacePath}"
+if (!CFG.isPullRequest) {
```

Comment:
I know that this code has been written by me but I am not sure if we should still do it like this. The problem is that builds running in branches might create and install artifacts in the cache that are broken and used by other builds. We should disable this at least for Maven plugin projects as well.

Reply:
Since we are solely running clean verify targets of maven there is currently no likely scenario of the cache being poluted by broken artifacts. For Maven deployments (e.g. Parent pom) we should not blindly copy the cache back.